context/brand-context/default/scss/00-tokens/_typography.variables.scss

```diff
@@ -0,0 +1,22 @@
+
+// Do not edit directly
+// Generated on Wed, 29 Jun 2022 21:08:16 GMT
+
+$tokens--typography-heading-font-family: serif;
+$tokens--typography-heading-letter-spacing: 0.2rem;
+$tokens--typography-heading-line-height: 1.2;
+$tokens--typography-heading-level-1-default-font-size: 2.5rem;
+$tokens--typography-heading-level-1-default-font-weight: 400;
+$tokens--typography-heading-level-1-fluid-font-size: clamp(1.875rem, 4vw, 2.5rem);
```

Comment:
What is the fallback for browsers not supporting clamp()? Browser default styles or do we have some sensible defaults?

Reply:
this isn't really part of this PR, as the design tokens were merged some time ago … 

however, the expectation is that these values are implemented (kind of) like [they are for Nature](https://github.com/springernature/frontend-toolkits/blob/master/context/brand-context/nature/scss/30-mixins/typography.scss#L15) (although, I'd suggest not as a variable in a placeholder that's used in a mixin, that's a bit Sass overkill). 

I guess the fallback would be [similar [to that in Springer Nature[(https://github.com/springernature/frontend-toolkits/blob/master/context/brand-context/springernature/scss/30-mixins/typography.scss#L3) where with no fallback provided … an non-supporting browser would (presumably) default to the user agent. 

This is why there is the `default` value and the `fluid` value - so there's a fallback. 

I'll try and work out how to document this to be more explicit (getting Tokens documentation genearated as part of the documentation on the Elements site would help)